### ISSUE TITLE
Pending doesn't function correctly

### ISSUE THREAD

**Issue by yxliang01** [nuclearpidgeon/SubjectMarks#2]
When I tried to add an assessment without filling in "Your result", it prompts the following error.
    Error!
    Please enter a valid assessment score

However, I don't see any other method to add assessment for which result is not out yet.

Anyway, this is so helpful! Better than using Excel, :P .

**Comment by nuclearpidgeon** [comment]
Hi Xiao, thanks for your feedback!

I last touched this app when I wrote it ~2 years ago as exam procrastination, so it's great to hear that it's still useful :).

Back when I wrote it, I think I had the idea of adding a 'pending' option for each assessment item, but probably just never got around to implementing it. What I used to do though was just put in all assessment items with a random guess for what I might get on the ones that weren't done yet, then just move the values up and down to see how what I would get on each item would affect my overall score.

If I have the time to look at this again I might see if I can add that 'pending' option though :) it would definitely be useful to be able to identify which assessment items had been completed or not.

Out of interest, how did find out about this app? I'm not a student anymore, so I'd be interested in ways to push this back out to current students (seeing as they're the only people that actually get use out of this!)

**Comment by yxliang01** [comment]
Right, exam procrastination is common at Unimelb. :P So, I have just done all the exams and homeworks yesterday (I am in US at this moment). Therefore, I spent some time on browsing Github. Then, suddenly think of an idea I had before.
I went to implement that simple idea (https://chrome.google.com/webstore/detail/take-me-lms-unimelb/ddchmncpocfpfelhaodfnnjeokbobgci). Search the keyword "unimelb" to see whether the new extension shows up. Then, I was thinking your ical would be helpful to me when I am back to Unimelb and I was a bit interested in how your ical extension works. Browse through its source code, see what you have done until I found this amazing app! 
I really wished I can get to know about this before!
Then, I showed this to my friend and realized that this has some missing function. (You know, we normally are so free right after exams, right? :P )
